pkg/lib/size/size.go

```diff
@@ -1,5 +1,7 @@
 package size
 
+// should this pkg just be dropped and made an exported function of the datastream pkg?
```

Comment:
Seems like we should move this into an exported function of the importer package, rather than having a full package (that imports the importer package anyway) to get the size of the endpoint.  Maybe there are intentions of adding more to this lib package?

Reply:
There was some pressure a while ago to provide an exportable package that CNV would import to determine the size of a remote file.  The intent being that front end automation would take be given an endpoint by a user, determine the size of the remote image, and generate a PVC just large enough to hold it.

Placing `size` in a standalone lib made sense then form an organizational standpoint, but I don't think it's being used in that way now.